packages/winston-log-enricher/lib/createFormatter.js

```diff
@@ -65,6 +65,21 @@ module.exports = function createFormatter(newrelic, winston) {
         .getOrCreateMetric(`Logging/lines/${levelLabel}`)
         .incrementCallCount()
     }
+
+    // if the agent has a log aggregator, add log lines to it.
```

Comment:
this comment is a tad misleading.  "if agent has log aggregator and within transaction context and the application logging forwarding feature is enabled, add log lines to it."

Reply:
There, I think I clarified this more.